pom.xml

```diff
@@ -551,7 +551,7 @@
 			<dependency>
 			    <groupId>org.glassfish.jaxb</groupId>
 			    <artifactId>jaxb-runtime</artifactId>
-			    <version>2.3.2</version>
+			    <version>3.0.0-M4</version>
```

Comment:
why upgrading to -M4?

Reply:
thanks @HerbertYiga  [here](https://search.maven.org/artifact/org.glassfish.jaxb/jaxb-runtime)  is the maven repository and from that page we can see that 3.0.0-M is the last version that the have .They have not yet release 3.0.0 version.